### PR TITLE
Added new users and schedules

### PR DESCRIPTION
**Pull request by adauzat90**
Additionally did some basic db optimizations

### CODE REVIEW

**Comment by henrywang1** [comment]
The change looks good, and we could develop new features more easily based on the dummy data you added.

There are two suggestions.
1. I did some research about ".each(&:save)" and I found that it could be improved more by warping it with a transaction and use "save!" to catch some errors from the database. 
Reference: [https://stackoverflow.com/questions/2292815/save-an-active-records-array](https://stackoverflow.com/questions/2292815/save-an-active-records-array)
2. Some print" and commented out code could be removed.

File: zlp-scheduler/db/seeds.rb [hunk 1]
```diff
@@ -6,6 +6,29 @@
...
+                @course = Course.where(class_hash)
+                @schedule.courses.push(@course)
+        end
+        
+        @schedule.save
+        @user.schedules.push(@schedule)   
+end
+
+
+@terms = Term.ImportTermList!
```
Comment (BWitemeyer) [inline zlp-scheduler/db/seeds.rb:23]: We really shouldn't be hitting the api (which this function does) in testing. And this seed data will probably be loaded every time a test is run (assuming I'm remembering how the testing stuff works correctly). So, we should probably just hard-code some individual terms and set one of them to be active.

File: zlp-scheduler/db/seeds.rb [hunk 2]
```diff
@@ -6,6 +6,29 @@
...
+@term = Term.find_by active: 1
+@term.opendate = DateTime.new(2001,2,3,4,5,6,'+03:00')
+@term.closedate = DateTime.new(2025,2,3,4,5,6,'+03:00')
+Subject.ImportSubjectsForTerm! @term
```
Comment (BWitemeyer) [inline zlp-scheduler/db/seeds.rb:27]: Again, this queries the api (see above comment).

File: zlp-scheduler/db/seeds.rb [hunk 3]
```diff
@@ -15,11 +38,118 @@
         @user.password = "Temp"
         @user.save
         
+@cohort = Cohort.new
+        @cohort.name = "Test Cohort"
+        @cohort.term_id = 202031
```
Comment (BWitemeyer) [inline zlp-scheduler/db/seeds.rb:43]: Is this creating a cohort that is attached to the current term? If so, why not use `@term.id`?

**Pull request closed without merging**